workers/loc.api/sync/data.inserter/data.checker/helpers/invert-sort.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports = (sortArr) => {
+  return sortArr.map(item => {
+    const _arr = [...item]
+
+    _arr[1] = item[1] > 0 ? -1 : 1
```

Comment:
Can explain this please, i dont understand the purpose of this.

Reply:
sure, we have the field for right order in the sync schema, for example:
https://github.com/bitfinexcom/bfx-reports-framework/blob/cd7568107951e65b14b6132a9b976b0ea09a31f1/workers/loc.api/sync/schema/index.js#L519
we use it when we fetch data from db
but when we want to sync starting point we should take the first row from db with inverted order by the sync schema
https://github.com/bitfinexcom/bfx-reports-framework/blob/cd7568107951e65b14b6132a9b976b0ea09a31f1/workers/loc.api/sync/data.inserter/data.checker/index.js#L191

Reply:
Ok thanks for clarifying